selenium/src/test/java/com/deque/html/axecore/selenium/Axe43xIntegrationTest.java

```diff
@@ -565,21 +629,28 @@ public void putsBackPageLoad() {
   public void withUnloadedIframes() {
     webDriver.get(fixture("/lazy-loaded-iframe.html"));
     String title = webDriver.getTitle();
-    AxeBuilder axeBuilder = new AxeBuilder().withRules(Arrays.asList("label", "frame-tested"));
+    AxeBuilder axeBuilder = new AxeBuilder().withOnlyRules(Arrays.asList("label", "frame-tested"));
     Results axeResults = axeBuilder.analyze(webDriver);
+    Capabilities caps = ((RemoteWebDriver) webDriver).getCapabilities();
+    Integer browserVersion = Integer.parseInt(caps.getBrowserVersion().split("\\.")[0]);
```

Comment:
We don't need to use the `RemoteWebDriver` here. We know we will always have a `ChromeDriver`. We can remove the remote webdriver test dependency and do:

```java
Capabilities capabilities = ((ChromeDriver) webDriver).getCapabilities();
int browserVersion = Integer.parseInt(capabilities.getBrowserVersion().split("\\.")[0]);
```